external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaContinuousStream.scala

```diff
@@ -46,7 +46,7 @@ import org.apache.spark.sql.util.CaseInsensitiveStringMap
  *                       properly read.
  */
 class KafkaContinuousStream(
-    offsetReader: KafkaOffsetReader,
+    private val offsetReader: KafkaOffsetReader,
```

Comment:
Is `private val` required for `Use CaseInsensitiveMap for KafkaOffsetReader`?

Reply:
Otherwise this code:
```
getField(stream, offsetReaderMethod)
```
throws exception something like:
```
java.lang.IllegalArgumentException: Can't find a private method named: offsetReader
```
The other possibility to add a getter explicitly which I thought is overkill.

Reply:
Another approach would be Scala reflection but definitely more verbose than explicit getter.